Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
I believe upstream changed the name from `lzma` to `xz` at some point, so our naming isn't that arbitrary. We could say that `libtiff` hasn't entirely caught up, but that's beside the point. The problem with `--with-lzma` is that this will make it harder for our dependency handling to handle the `xz` dependency and why we strongly prefer `depends_on "xz" => :optional` over `depends_on "xz" if <condition>`. `brew options libtiff` will help with finding the correct option name.

That said, is there a strong argument to be made against enabling LZMA support by default in `libtiff`? If not, I'd prefer we do just that and avoid all complications the option and dependency handling creates. (And provide a more feature-complete `libtiff` by default.)